testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxSetup.java

```diff
@@ -58,19 +61,23 @@
 @VisibleForTesting
 final class BlackBoxSetup {
 
+    private final BoundedContextName contextName;
```

Comment:
If the one above resolved, this field requires (maybe) both renaming and explanation OR removal.